content/turing-machines/machines-computations/unary-numbers.tex

```diff
@@ -27,7 +27,9 @@
 \[
 \TMstroke^{k_1} \TMblank \TMstroke^{k_2} \TMblank \dots \TMblank \TMstroke^{k_n}
 \]
-with output $\TMstroke^{f(k_1, \dots, k_n)}$.
+with output $\TMstroke^{f(k_1, \dots, k_n)}$. 
```

Comment:
The problem with the current definition in the text is that to be able to combine two turing machines the second one needs to recognize if the tape is empty or not (the first machine outputs 0 iff the tape is empty). For example, I think the proof that the function s isn't Turing computable is problematic since we can't assume that the output is written directly to the right of the TMendtape symbol.

Reply:
Excellent point. See issue #191